platform/mellanox/rules.mk

```diff
@@ -30,6 +30,7 @@ include $(PLATFORM_PATH)/issu-version.mk
 include $(PLATFORM_PATH)/mlnx-onie-fw-update.mk
 include $(PLATFORM_PATH)/mlnx-ssd-fw-update.mk
 include $(PLATFORM_PATH)/install-pending-fw.mk
+include $(PLATFORM_PATH)/iproute2.mk
```

Comment:
Can you add this after line `include $(PLATFORM_PATH)/mlnx-ssd-fw-update.mk` will otherwise cause an unnecessary merge conflict with my other pr `https://github.com/sonic-net/sonic-buildimage/pull/14594`

Reply:
Done.